nitorch/tools/img_statistics.py

```diff
@@ -223,4 +225,7 @@ def estimate_noise(dat, show_fit=False, fig_num=1, num_class=2,
     mu_not_noise = sum(w * mu1)
     sd_not_noise = sum(w * sd1)
 
-    return sd_noise, sd_not_noise, mu_noise, mu_not_noise
+    if chi:
```

Comment:
If I do this, do you have a preference for how to approach `sd0, sd1, mu0, mu1, dof0 = estimate_noise(dat, chi=chi)` in preproc()? I think constant number of outputs is recommended.

Reply:
I write a lot of functions with variable numbers of outputs. But you're right, it's probably a very bad idea!
You could refactor the function so that it returns `prm_noise, prm_not_noise`, where each of these variables is a dictionary with fields 'sd', 'mean' and optionally 'dof' (in the Chi case)?

This would break any existing call to the function that assume that it returns `sd0, sd1, mu0, mu1` tough (but so does adding a fifth output). I think that Mikael uses this function in his own code. @brudfors, do you mind if we do this change (= return two dictionaries instead of four scalars)?

Reply:
changed to dictionary output, left variable output commented out both here and in preproc(), not sure which one is to stay